R/utils.R

```diff
@@ -17,24 +17,47 @@ formatSolveBioResponse <- function (res, raw = FALSE) {
         res$'_url' = url
         return(res)
     }
+
 }
 
-formatSolveBioQueryResponse <- function (res, raw = FALSE, row.names = NULL) {
+formatSolveBioQueryResponse <- function (id, res, raw = FALSE, row.names = NULL) {
```

Comment:
Lets change to col.names instead of row.names as that is more accurate

Reply:
Done.